fmt/testdata/ReturnStatement/original.sol

```diff
@@ -13,8 +13,7 @@ contract ReturnStatement {
     return /* return empty 2 */ ; // return empty 3
         }
 
-        /* return empty 4 */ return 
-            // return empty 5
+        /* return empty 4 */ return // return empty 5
         ;
```

Comment:
wdyt about formatting after semicolon for empty return statements?
```
/* return empty 4 */ return; // return empty 5
```

Reply:
yeah makes sense 👍

Reply:
I just took a look @rkrasiuk and that does seem to be the behavior right now https://github.com/jpopesculian/foundry/blob/fmt-bug-block-comments/fmt/testdata/ReturnStatement/fmt.sol#L17 . To be honest I changed the test here because the `// return empty 5` was being matched as a prefix after my changes and translating to

```
/* return empty 4 */
return;
// return empty 5
```

which I thought was actually correct behavior, but also so weird that why would we test for it...